sdk/storage/storage-file-datalake/package.json

```diff
@@ -5,7 +5,7 @@
   "sdk-type": "client",
   "main": "./dist/index.js",
   "module": "./dist-esm/src/index.js",
-  "types": "./typings/src/index.d.ts",
+  "types": "typings/storage-datalake.d.ts",
```

Comment:
Which config change generates `storage-blob.d.ts` files?

Reply:
It's specified in `api-extractor.json`.  I will update it to have `file` in the filename.

Reply:
updated

Reply:
Fixed